td/telegram/StickersManager.cpp

```diff
@@ -2744,13 +2742,13 @@ tl_object_ptr<telegram_api::inputStickerSetItem> StickersManager::get_input_stic
         point, sticker->mask_position_->x_shift_, sticker->mask_position_->y_shift_, sticker->mask_position_->scale_);
   }
 
-  int32 flags = 0;
   if (mask_coords != nullptr) {
-    flags |= telegram_api::inputStickerSetItem::MASK_COORDS_MASK;
+    int32 flags = telegram_api::inputStickerSetItem::MASK_COORDS_MASK;
+    return make_tl_object<telegram_api::inputStickerSetItem>(flags, std::move(input_document), sticker->emojis_,
+                                                           std::move(mask_coords));
   }
 
-  return make_tl_object<telegram_api::inputStickerSetItem>(flags, std::move(input_document), sticker->emojis_,
-                                                           std::move(mask_coords));
+  return -1;
```

Comment:
This shouldn't compile, -1 isn't convertible to unique_ptr.

Reply:
@levlam I'm sorry, I looked at the lambda above. But here it is necessary to return an invalid state.